sdk/openai/Azure.AI.OpenAI/tests/CompletionsTests.cs

```diff
@@ -108,7 +108,6 @@ public async Task AdvancedCompletionsOptions(Service serviceTarget)
                 GenerationSampleCount = 3,
                 Temperature = 0.75f,
                 User = "AzureSDKOpenAITests",
-                Echo = true,
```

Comment:
was this removal due to the `instruct` model switch no longer supporting the property? I think that's fine if so; the schema still supports `echo` and it'd be nice if we still validated it, but given this is "legacy" status I think that can be deferred

Reply:
Yeah, setting 'echo' and 'logprobs' at the same time is not supported for `instruct` model, so I seperate it into a new test `AdvancedCompletionsOptionsWithEcho`